File: solutions/julia/sieve/1/sieve.jl
function sieve(limit)
    allns = [n for n=1:limit]
        for cur in 2:limit
             if  allns[cur]!=0 
                next_prime = cur
                k = 2*next_prime
                while k<=limit
                    allns[k] = 0
                    k += next_prime
                end
             end
         end 
    allns[allns .> 1]
end

File: solutions/python/sieve/1/sieve.py
def primes(limit):
    allns = [n for n in range(1,limit+1)]
    for current in range(2,limit+1):
        if allns[current-1]!=0:
            next_prime = current
            k = 2*next_prime
            while k<=limit:
                allns[k-1] = 0
                k += next_prime
    return [p for p in allns if p>1]

